### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD028.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#31] [General]
# https://blog.3place.cn/3place/fsd/FSD028.html

https://blog.3place.cn/3place/fsd/FSD028.html

**Comment by picklecai** [comment]
矮小而有大志的妖精，急忙出来扶危济困；
对不幸的人俱表同情，
不分邪恶还是神圣。

缺少判断力时，就是“弱者有理”。

**Comment by picklecai** [comment]
轻风带着微微的凉意，
充满绿荫四合的原野，
黄昏送来甘美的香气，
垂下她那薄雾的罗帏。
低声呼唤甘美的安宁，
让心儿像孩子般睡稳；

好美呀，仙境也不过如此。何时入仙境？

**Comment by picklecai** [comment]
众生往往犹豫不定；
大丈夫事事都能实现，
因为他能知而即行。

这句话好像在说“性格决定命运”。

**Comment by picklecai** [comment]
日神的车轮辚辚滚来，
光的声响多么惊人！

如果是观音，就当如此。不对，是听光，就当如此。再没有比太阳光更巨大的声响了。

**Comment by picklecai** [comment]
大地啊，你在昨夜也毫无变动，
如今在我脚下又焕然一新，
你已开始用欢乐将我包围，
你鼓励我，唤起我坚强的决心，
使我努力追求最高的存在。

和上一部开头比起来，这时的浮士德是个完完全全的新人了。这是拿一个姑娘的青春和生命换来的变化。生命要有所经历，才能称其为什么，才能成其为生命。感谢，npc。

**Comment by picklecai** [comment]
巨人似的山峰
已宣告最最庄严的时刻到来；
永恒的天光首先让它们享用，
然后才下临到我们的头上。

这个视角，好有趣。

**Comment by picklecai** [comment]
人总是这样，当我们满怀着希望，
自信地快要完遂最高的心愿，
看到成功的大门豁然开敞；
可是，从那永恒的深处却喷出
无数的火焰，我们感到多惊慌；

这是什么情况？

**Comment by picklecai** [comment]
那穿过岩隙奔腾直下的瀑布，
使我越看越欣喜若狂。
它一叠一叠地翻滚，化成千股，
然后又分作千万道急流奔涌，
向空中喷溅出无数飞沫细珠。

这个说得我想去看瀑布。

**Comment by picklecai** [comment]
彩虹反映出人类的努力上进。
细心揣摩，你就会更加领悟；
要从多彩的映象省识人生。

努力上进皆为虚妄？
